node/src/chaincodeevent.ts

```diff
@@ -39,7 +39,7 @@ export interface ChaincodeEvent {
 }
 
 export function newChaincodeEvents(responses: CloseableAsyncIterable<ChaincodeEventsResponse>): CloseableAsyncIterable<ChaincodeEvent> {
-    return {
+    return  {
```

Comment:
Changes to this file are only accidental (and incorrect) formatting. I suggest reverting this file to its unmodified state.